packages/nimble-components/src/patterns/dropdown/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import {
+    controlHeight as controlHeightToken,
+    smallPadding
+} from '../../theme-provider/design-tokens';
+
+const maxVisibleOptions = 10.5;
+
+export function getDropdownMaxHeight(
```

Comment:
The `loading-visible` attribute just went in. How should the presence of the loading item impact this height? Since it's not scrollable I'd vote to treat it similar to the search field, and have it add a fixed amount to the height.